docs/pages/blog/mui-v5.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Announcing MUI Core v5.0
```

Comment:
```suggestion
title: Introducing MUI Core v5.0
```

How about it? I feel like it's more usual to use _Introducing_ than _Announcing_ whenever marketing a new product/whatever. Just a suggestion.

Reply:
Ok done